content/guides/outliers.md

```diff
@@ -57,11 +57,11 @@ You can also define a monitor to alert when an outlier is detected in an importa
 
 {{< img src="outliers/outliers-alert-snapshot.png" >}}
 
-For example, to alert when a Cassandra host is abnormally loaded compared to the rest of the group, you can [add a new outlier monitor](https://app.datadoghq.com/monitors#create/algorithm) for the metric:
+For example, to alert when a Cassandra host is abnormally loaded compared to the rest of the group, you can [add a new outlier monitor](https://app.datadoghq.com/monitors#create/outlier) for the metric.
 
-{{< img src="outliers/outliers-new-monitor-define-metric.png" >}}
+Start by navigating to the [New Monitor](https://app.datadoghq.com/monitors#/create) page and selecting Outlier. The "Outlier Detection" option in step (1) should already be selected. Define the metric in step (2) just like you would for any other metric alert.
```

Comment:
Actually... I think let's delete this sentence: "The Outlier Detection option in step 1 should already be selected". It doesn't convey any useful information, and also "step (1)" can be mistakenly taken to mean step (1) above (i.e. **Show Outliers in Dashboards or Screenboards**).

Then let's remove reference to step (2), as well: "Define the metric just like you would for any other metric alert".